### ISSUE TITLE
Collection over Religious Holidays

### ISSUE THREAD

**Issue by grburgess** [grburgess/awesome-reading-group#8]
### Simple and statistically sound strategies for analysing physical theories

https://arxiv.org/abs/2012.09874

Physical theories that depend on many parameters or are tested against data from many different experiments pose unique challenges to parameter estimation. Many models in particle physics, astrophysics and cosmology fall into one or both of these categories. These issues are often sidestepped with very simplistic and statistically unsound ad hoc methods, involving naive intersection of parameter intervals estimated by multiple experiments, and random or grid sampling of model parameters. Whilst these methods are easy to apply, they exhibit pathologies even in low-dimensional parameter spaces, and quickly become problematic to use and interpret in higher dimensions. In this article we give clear guidance for going beyond these rudimentary procedures, suggesting some simple methods for performing statistically sound inference, and recommendations of readily-available software tools and standards that can assist in doing so. Our aim is to provide physicists with recommendations for reaching correct scientific conclusions, with only a modest increase in analysis burden.

#### Comments:
3ML

**Comment by grburgess** [comment]
### Probing multiple populations of compact binaries with third-generation gravitational-wave detectors

https://arxiv.org/abs/2012.09876

Third-generation (3G) gravitational-wave (GW) detectors will be able to observe binary-black-hole mergers (BBHs) up to redshift of ∼30. This gives unprecedented access to the formation and evolution of BBHs throughout cosmic history. In this paper we consider three sub-populations of BBHs originating from the different evolutionary channels: isolated formation in galactic fields, dynamical formation in globular clusters and mergers of black holes formed from Population III (Pop III) stars at very high redshift. Using input from populations synthesis analyses, we created two months of simulated data of a network of 3G detectors made of two Cosmic Explorers and an Einstein Telescope, consisting of ∼16000 field and cluster BBHs as well as ∼400 Pop III BBHs. First, we show how one can use non-parametric models to infer the existence and characteristic of a primary and secondary peak in the merger rate distribution. In particular, the location and the height of the secondary peak around z≈12, arising from the merger of Pop III remnants, can be constrained at (10%) level. Then we perform a modeled analysis, using phenomenological templates for the merger rates of the three sub-population, and extract the branching ratios and the characteristic parameters of the merger rate densities of the individual formation channels. With this modeled method, the uncertainty on the measurement of the fraction of Pop III BBHs can be improved to ≲10%, while the ratio between field and cluster BBHs can be measured with an uncertainty of ∼50%.

**Comment by cescalara** [comment]
### What role will binary neutron star merger afterglows play in multimessenger cosmology?

https://arxiv.org/abs/2012.12836

Binary neutron star mergers offer a new and independent means of measuring the Hubble constant H0 by combining the gravitational-wave inferred source luminosity distance with its redshift obtained from electromagnetic follow-up. This method is limited by intrinsic degeneracy between the system distance and orbital inclination in the gravitational-wave signal. Observing the afterglow counterpart to a merger can further constrain the inclination angle, allowing this degeneracy to be partially lifted and improving the measurement of H0. In the case of the binary neutron star merger GW170817, afterglow light-curve and imagery modeling thus allowed to improve the H0 measurement by a factor of 3. However, systematic access to afterglow data is far from guaranteed. In fact, though each one allows a leap in H0 precision, these afterglow counterparts should prove rare in forthcoming multimessenger campaigns. We combine models for emission and detection of gravitational-wave and electromagnetic radiation from binary neutron star mergers with realistic population models and estimates for afterglow inclination angle constraints. Using these models, we quantify how fast H0 will be narrowed-down by successive multimessenger events with and without the afterglow. We find that, because of its rareness and though it greatly refines angle estimates, the afterglow counterpart should not significantly contribute to the measurement of H0 in the long run.

**Comment by anastasia-tsvetkova** [comment]
### **A bright γ-ray flare interpreted as a giant magnetar flare in NGC 253**

https://www.nature.com/articles/s41586-020-03076-9 or https://arxiv.org/pdf/2101.05104.pdf

Soft γ-ray repeaters exhibit bursting emission in hard X-rays and soft γ-rays. During the active phase, they emit random short (milliseconds to several seconds long), hard-X-ray bursts, with peak luminosities of 10^36 to 10^43 erg per second. Occasionally, a giant flare with an energy of around 10^44 to 10^46 erg is emitted. These phenomena are thought to arise from neutron stars with extremely high magnetic fields (10^14 to 10^15 gauss), called magnetars. A portion of the second-long initial pulse of a giant flare in some respects mimics short γ-ray bursts, which have recently been identified as resulting from the merger of two neutron stars accompanied by gravitational-wave emission. Two γ-ray bursts, GRB 051103 and GRB 070201, have been associated with giant flares. Here we report observations of the γ-ray burst GRB 200415A, which we localized to a 20-square-arcmin region of the starburst galaxy NGC 253, located about 3.5 million parsecs away. The burst had a sharp, millisecond-scale hard spectrum in the initial pulse, which was followed by steady fading and softening over 0.2 seconds. The energy released (roughly 1.3 × 10^46 erg) is similar to that of the superflare from the Galactic soft γ-ray repeater SGR 1806−20 (roughly 2.3 × 10^46 erg). We argue that GRB 200415A is a giant flare from a magnetar in NGC 253.

**Comment by grburgess** [comment]
## How not to obtain the redshift distribution from probabilistic redshift estimates: Under what conditions is it not inappropriate to estimate the redshift distribution N(z) by stacking photo-z PDFs?

https://arxiv.org/abs/2101.04675


The scientific impact of current and upcoming photometric galaxy surveys is contingent on our ability to obtain redshift estimates for large numbers of faint galaxies. In the absence of spectroscopically confirmed redshifts, broad-band photometric redshift point estimates (photo-zs) have been superseded by photo-z probability density functions (PDFs) that encapsulate their nontrivial uncertainties. Initial applications of photo-z PDFs in weak gravitational lensing studies of cosmology have obtained the redshift distribution function (z) by employing computationally straightforward stacking methodologies that violate the laws of probability. In response, mathematically self-consistent models of varying complexity have been proposed in an effort to answer the question, "What is the right way to obtain the redshift distribution function (z) from a catalog of photo-z PDFs?" This letter aims to motivate adoption of such principled methods by addressing the contrapositive of the more common presentation of such models, answering the question, "Under what conditions do traditional stacking methods successfully recover the true redshift distribution function (z)?" By placing stacking in a rigorous mathematical environment, we identify two such conditions: those of perfectly informative data and perfectly informative prior information. Stacking has maintained its foothold in the astronomical community for so long because the conditions in question were only weakly violated in the past. These conditions, however, will be strongly violated by future galaxy surveys. We therefore conclude that stacking must be abandoned in favor of mathematically supported methods in order to advance observational cosmology.

### Comments:

Stacking is bad.
